README.md

```diff
@@ -78,10 +78,9 @@ Your submission must be compliant with the GRASS
 submission rules as found in the GRASS source code
 and [RFC2 (Legal aspects of submission)](https://github.com/OSGeo/grass/blob/main/doc/development/rfc/legal_aspects_of_code_contributions.md).
 
-Also please read submitting instructions before creating a pull request:
+Also, please read the general GRASS contributing instructions before creating a pull request:
+<https://github.com/OSGeo/grass/?tab=readme-ov-file#contributing>
```

Comment:
Any reason not to link https://github.com/OSGeo/grass/blob/main/CONTRIBUTING.md directly?